modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportImportObjectEntriesWithSiteScope.testcase

```diff
@@ -0,0 +1,240 @@
+			var scope = TestCase.getSiteName();
+
+			ImportExport.replaceAttachmentIdAndHrefInFile(
+				en_US_plural_label = "students",
+				fileName = "multiple_site_scope_objectEntries_import_create_update.jsonl",
+				scopeKey = "true");
+
+			ImportExport.importFile(
+				entityType = "C_Student (v1_0 - Liferay Object REST)",
+				fileName = "multiple_site_scope_objectEntries_import_create_update.jsonl",
+				scope = ${scope});
+		}
+
+		task ("Then one new Student item is created") {
+			var response = ObjectDefinitionAPI.getObjectEntries(
+				en_US_plural_label = "students",
+				scopeKey = "true");
+
+			var actualValue = JSONUtil.getWithJSONPath(${response}, "$..items[?(@.externalReferenceCode=='testErc')].name");
+
+			TestUtils.assertEquals(
+				actual = ${actualValue},
+				expected = "Jack");
+		}
+
+		task ("And Then existing Student item's name is updated") {
+			var actualValue = JSONUtil.getWithJSONPath(${response}, "$..items[?(@.externalReferenceCode=='studentErc')].name");
+
+			TestUtils.assertEquals(
+				actual = ${actualValue},
+				expected = "John");
+		}
+
+		task ("And Then existing Student's relationship field is unchanged") {
+			var actualValue = JSONUtil.getWithJSONPath(${response}, "$..items[?(@.externalReferenceCode=='studentErc')].subjectStudentsERC");
+
+			TestUtils.assertEquals(
+				actual = ${actualValue},
+				expected = "Math");
+		}
+	}
+
+	@priority = 4
+	test CanCreateSiteScopedObjectEntryFromFileContainingExistingAndNewWithUncheckedStopImportOnError {
+		property portal.acceptance = "true";
+
+		task ("When importing .json file as 'C_Student' Entity Type with 'Only Add New Records Import Strategy', 'Stop the Import on Error' unchecked") {
+			var scope = TestCase.getSiteName();
+
+			ImportExport.importFile(
+				entityType = "C_Student (v1_0 - Liferay Object REST)",
+				fileName = "multiple_site_scope_objectEntries_import_create.jsonl",
+				importStrategy = "Only Add New Records",
+				scope = ${scope},
+				stopImportOnError = "false");
+		}
+
+		task ("Then one new Student item is created") {
+			var response = ObjectDefinitionAPI.getObjectEntries(
+				en_US_plural_label = "students",
+				scopeKey = "true");
+
+			var actualValue = JSONUtil.getWithJSONPath(${response}, "$.items[1].name");
+
+			TestUtils.assertEquals(
+				actual = ${actualValue},
+				expected = "Jack");
+		}
+	}
+
+	@priority = 4
+	test CannotCreateSiteScopedObjectEntryWithoutRequiredFields {
+		property portal.acceptance = "true";
+
+		task ("When importing .json file as 'C_Student' Entity Type with default setting") {
+			var scope = TestCase.getSiteName();
+
+			ImportExport.importFile(
+				entityType = "C_Student (v1_0 - Liferay Object REST)",
+				fileName = "site_scope_objectEntry_import_without_required_fields.json",
+				noImport = "true",
+				scope = ${scope});
+
+			Button.click(button = "Next");
+		}
+
+		task ("Then importing the item is prevented") {
+			AssertElementNotPresent(
```

Comment:
Wondering is there any error message/alert?

Reply:
what your said it's another test 'ErrorMessageIsShownWhenRequiredFieldsMissing' FP=3

Reply:
got it, thanks